pkg/vfs/internal.go

```diff
@@ -155,9 +155,10 @@ func handleInternalMsg(ctx Context, cmd uint32, r *utils.Buffer) []byte {
 	case meta.Info:
 		var summary meta.Summary
 		inode := Ino(r.Get64())
+		recursive := r.Get8()
```

Comment:
IF the client is older version, there will be no extra byte .

Reply:
459c3bc